ukaea-schema/diagnostics/camera.schema.json

```diff
@@ -3,14 +3,14 @@
     "$id": "urn:ukaea:experiment:hive:diagnostics:camera-schema",
     "description": "Metadata to describe camera diagnostics",
     "type": "object",
-    "required": ["camera_information", "lens_information", "capture_settings"],
+    "required": ["cameraInformation", "lensN", "captureSettings"],
```

Comment:
Should this be "lensInformation"?

Reply:
Yes, 
Nathan. I have changed it now. I will add the MR